dashboard/app/models/user.rb

```diff
@@ -333,6 +333,8 @@ def district_teachers(cohort = nil)
 
   before_create :generate_secret_words
 
+  before_create :clear_ui_tips
```

Comment:
nit: This naming seems odd to me. It doesn't make sense to "clear" something on a new user.

How about something like `suppress_ui_tips_for_new_users`, which is more self-explanatory?